app/services/model/RejectionReason.scala

```diff
@@ -61,10 +54,17 @@ object RejectionReason {
       notification.errors.map { error =>
         RejectionReason(
           error.validationCode,
-          getErrorDescription(error.validationCode),
+          getCdsErrorDescription(error.validationCode),
```

Comment:
Couldn’t we keep the model as it was and do the if toggle here?

Reply:
Keeps the logic in Scala code

Reply:
Very good point!!!!

Reply:
I have just discussed this with @PatrykRudnicki.
We would have to create a new `RejectionReasonViewModel` and possibly a new `RejectionReasonService`, since it would not be a good design to inject the AppConfig into the `RejectionReason`.
After discussing other approaches it became clear that all of them had advantages and disadvantages. 
I agree with the conditionality leaking into the view though, but I cannot find a compelling enough solution to do it in another way.